static/js/country_cards.js

```diff
@@ -183,6 +190,65 @@ const setSearch = (event) => {
     selectBtn.firstElementChild.innerText = 'Filter by Region';
 }
 
+const changeGridLayout = (event) => {
+    // Make sure target is always the button
+    const btn = event.target.closest('button');
+
+    switch (btn.classList[1]) {
+        case 'tile-btn-1':
+            countryCardContainer.style.gridTemplateColumns = 'repeat(1, minmax(280px, 1fr))';
+            break;
+        case 'tile-btn-2':
+            countryCardContainer.style.gridTemplateColumns = 'repeat(2, minmax(280px, 1fr))';
+            break;
+        case 'tile-btn-3':
+            countryCardContainer.style.gridTemplateColumns = 'repeat(3, minmax(280px, 1fr))';
+            break;
+        case 'tile-btn-4':
+            countryCardContainer.style.gridTemplateColumns = 'repeat(4, minmax(280px, 1fr))';
+            break;
+    }
+}
+
+const resetGridLayout = (event) => {
+    const window = event.target;
+
+    const smallScreen = window.innerWidth < 720
+    const mediumScreen = window.innerWidth >= 720 && window.innerWidth < 1040
+    const largeScreen = window.innerWidth >= 1040 && window.innerWidth < 1360
+    const xtraLargeScreen = window.innerWidth >= 1360
+
+    const tileLayout1 = countryCardContainer.style.gridTemplateColumns === 'repeat(1, minmax(280px, 1fr))';
+    const tileLayout2 = countryCardContainer.style.gridTemplateColumns === 'repeat(2, minmax(280px, 1fr))';
+    const tileLayout3 = countryCardContainer.style.gridTemplateColumns === 'repeat(3, minmax(280px, 1fr))';
+    const tileLayout4 = countryCardContainer.style.gridTemplateColumns === 'repeat(4, minmax(280px, 1fr))';
+    // If amount of tiles exceeds the window width, remove the inline styling placed by the tile-btn.
+    // Or if the amount of tiles does not correspond with the amount of tiles that can be chosen from, also remove the inline styling
+
+    if (smallScreen) {
```

Comment:
Denk dat het ook netter is om zulke css and media queries zo veel mogelijk bij css te houden.